### PR TITLE
fix(tests): User report test clear django cache

### PR DESCRIPTION
**Pull request by Zylphrex**
EventUser is stored in postgres but EventManager will not recreate the EventUser if the cache key exists. Make sure to clear the cache before running this test or we may hit a situation where the event user exists in the cache but not in postgres.

### CODE REVIEW

File: src/sentry/utils/pytest/fixtures.py [hunk 1]
```diff
@@ -442,3 +443,8 @@ def inner(key, value):
         request.addfinalizer(lambda: ctx_mgr.__exit__(None, None, None))
 
     return inner
+
+
+@pytest.fixture
+def django_cache():
```
Comment (jjbayer) [inline src/sentry/utils/pytest/fixtures.py:449]: nit: Should we call this `clear_django_cache` to clarify what it does? For example, when I have fixtures `redis_cache` and `django_cache`, I would expect that both are objects I can interact with.

File: src/sentry/utils/pytest/fixtures.py [hunk 2]
```diff
@@ -442,3 +443,8 @@ def inner(key, value):
...
+    cache.clear()
```
Comment (jjbayer) [inline src/sentry/utils/pytest/fixtures.py:450]: Will this break other tests when multiple tests run in parallel?

**Review by jjbayer — COMMENTED** [review]
@Zylphrex thanks for looking into this!

File: src/sentry/utils/pytest/fixtures.py (hunk 1 above)
Comment (Zylphrex) [reply src/sentry/utils/pytest/fixtures.py:449]: I've updated this fixture to yield the cache object so it's interactable now.

File: src/sentry/utils/pytest/fixtures.py (hunk 2 above)
Comment (Zylphrex) [reply src/sentry/utils/pytest/fixtures.py:451]: AttributeError: 'BoundLoggerFilteringAtNotset' object has no attribute 'log'

**Review by jjbayer — APPROVED** [review]

**Pull request merged**